.changeset/nine-colts-raise.md

```diff
@@ -0,0 +1,6 @@
+---
+'@compiled/react': patch
+---
+
+- Give a name to CSS var used for empty values
```

Comment:
curious to know why we need a name

Reply:
Current Chrome will treat rules using empty CSS var names as invalid. This is also why new Loki version was breaking on VR, because it contained new Chromium :)